docs/dev-guide/contributing/dev_setup.rst

```diff
@@ -64,19 +68,25 @@ Follow these steps:
       $ sudo systemctl enable nfs-server && sudo systemctl start nfs-server
 
 #. You will need to allow NFS services through your firewall::
-   
+
       $ sudo firewall-cmd --permanent --add-service=nfs
       $ sudo firewall-cmd --permanent --add-service=rpc-bind
       $ sudo firewall-cmd --permanent --add-service=mountd
       $ sudo firewall-cmd --reload
 
+.. note::
+
+  It might happen that Vagrant won't start due to the
```

Comment:
This can be rephrased to something like

"If Vagrant will not start due to the error message ...., you can solve it by using ...."